packages/block-editor/src/components/block-tag-selection/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarDropdownMenu } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import TagIcon from './tag-icon';
+
+// Default HTML tags
+const DEFAULT_TAGS = [ 'h1', 'h2', 'h3', 'h4', 'h5', 'h6' ];
+
+const POPOVER_PROPS = {
+	className: 'block-library-heading-level-dropdown',
```

Comment:
A more accurate className should be used here.
Should the old classname be kept for backwards compatibility or can it be removed?